docs/pages/languages.mdx

```diff
@@ -3,6 +3,39 @@
 Languages are essentially Node.js modules that encapsulate how to retrieve
 and create content. You can think of them as **small edge functions** that are executed on the Agents device and that can communicate with different backends and technologies.
 
+## Why Deno Compatibility?
+
+Deno compatibility is required because of its several advantages:
```

Comment:
```suggestion
Deno compatibility is required because Languages get executed inside a sandbox that the ADAM executor spawns, and since it needs access to internals of ADAM, this needs to be an integrated JavaScript interpreter that is intertwined with the ADAM implementation. We've decided to build this Language engine on Deno of its several advantages:
```